src/services/createCurrentRoute.ts

```diff
@@ -18,7 +19,7 @@ export function createCurrentRoute(fallbackRoute: ResolvedRoute, push: RouterPus
   const route = reactive({ ...fallbackRoute })
 
   const updateRoute: ResolvedRouteUpdate = (newRoute) => {
-    Object.assign(route, { ...newRoute })
+    Object.assign(route, { [isRejectionRouteSymbol]: false, ...newRoute })
```

Comment:
nit
```suggestion
    Object.assign(route, { 
      [isRejectionRouteSymbol]: false,
      ...newRoute 
    })
```

Reply:
Any chance this fixes https://github.com/kitbagjs/router/issues/244?

Reply:
@pleek91 just updated my test branch in router-preview to use this version of router and the issue seems to still exist unfortunately 😞